src/main/java/uk/ac/ebi/biosamples/jsonschemastore/ena/SchemaTemplateGenerator.java

```diff
@@ -27,138 +26,143 @@
 @Component
 @RequiredArgsConstructor
 public class SchemaTemplateGenerator {
-    private static final ObjectMapper mapper = new ObjectMapper();
-    private final SchemaObjectPopulator populator;
+  private static final ObjectMapper mapper = new ObjectMapper();
 
-    public static String getTaxonTemplate() {
-        return getJsonString(Map.of("isValidTaxonomy", true));
-    }
+  static {
+    mapper.configure(JsonReadFeature.ALLOW_UNESCAPED_CONTROL_CHARS.mappedFeature(), true);
+  }
```

Comment:
The more "spring" way would be to define it as a bean that would be a dependency of this component. There is already an ObjectMapper bean defined in AuthConfig, so if you ned a special one for the export scenario, you can use a `@Qualifier`

```java
@Bean 
public ObjectMapper objectMapper() {
    ObjectMapper mapper = new ObjectMapper();
    mapper.configure(
      JsonReadFeature.ALLOW_UNESCAPED_CONTROL_CHARS.mappedFeature(), 
      true);
  return mapper;
}
```